### PR TITLE
Edit a Location's field except phone number 

### PR DESCRIPTION
**Pull request by dwhenry**
Add a show and edit pages for location. 

This also removes updating of the visibility from the locations directory.

### CODE REVIEW

File: app/views/admin/locations/_location.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<table class="table">
+  <tr>
+    <td class="location-details-table__fixed-width-column">Address:</td>
+    <td class="t-address"><%= location.address.to_a.join('<br>').html_safe %></td>
```
Comment (benbarnett) [inline app/views/admin/locations/_location.html.erb]: Didn't we have a helper somewhere for this?

**Comment by benbarnett** [comment]
Pull requests should have some context around what the change is making and why. Just giving it a title of 'Editing' doesn't provide much information.

As discussed on Friday, the only comment I have is the one I've put here. My understanding is that you wanted to improve the code from a more general perspective, which was left up to you. Let me know if you would like a re-review.

**Comment by dwhenry** [comment]
@benbarnett - Have added a description to the PR, this is something I am working to do to all PR and sorry I missed this one out.

I have now finished refactoring the code and would appreciate any feedback.

**Comment by benbarnett** [comment]
:+1:

**Pull request merged**